SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityCleaner.java

```diff
@@ -458,6 +462,24 @@ private TaskCleanupType pause(SingularityRequestCleanup requestCleanup, Iterable
     return cleanupType;
   }
 
+  private void delete(SingularityRequestCleanup requestCleanup, Iterable<SingularityTaskId> activeTaskIds){
```

Comment:
@ssalinas I kept the name as delete since it followed the pattern set by the other actions (pause, bounce, etc.)